src/main/java/data/Account.java

```diff
@@ -15,22 +15,22 @@
 
 /** A class representing a billing account object. */
 public class Account {
-    private String accountID;
-    private String vendorID;
-    private String entity;
-    private String currency;
-    private String direction;
-    private String legacyAccountID;
-    private int nextGenAccountID;
-    private String matchingMode;
-    private String aggregationMode;
-
-    public Account(String accountID, String vendorID, String entity, 
-        String currency, String direction, String legacyAccountID, 
-        int nextGenAccountID, String matchingMode, String aggregationMode) {
-
-        this.accountID = accountID;
-        this.vendorID = vendorID;
+  private String accountID;
+  private String vendorID;
+  private String entity;
+  private String currency;
+  private String direction;
+  private String legacyAccountID;
+  private int nextGenAccountID;
+  private String matchingMode;
+  private String aggregationMode;
+
+  public Account(String accountID, String vendorID, String entity, 
+    String currency, String direction, String legacyAccountID, 
+    int nextGenAccountID, String matchingMode, String aggregationMode) {
+
```

Comment:
Should the setters be private for this class or are we planning on them being used outside of this class?

Reply:
I created setter methods because I thought that you might want to use them. I currently don't use them, so I could delete them if you have no need for them.